app/src/main/java/com/example/ncov19traking/NCoVRecyclerAdapter.kt

```diff
@@ -61,22 +67,22 @@ class NCoVRecyclerAdapter : RecyclerView.Adapter<NCoVRecyclerAdapter.ViewHolder>
 
     override fun getFilter() = countryFilter
 
-    private var countryFilter = object : Filter(){
+    private var countryFilter = object : Filter() {
```

Comment:
We should always keep an order in the classes we create, there is no clear definition for this but one I like to use is , 

- private variables
-public variables
-init method/ constructors
-public methods
-private methods
-companion object.

Of course, this could vary from project to project but you should always try to have order.

Let's order things in this class.

Reply:
done